Ruby/Ruby.sublime-syntax

```diff
@@ -68,7 +68,7 @@ file_extensions:
   - thor
   - Thorfile
   - Vagrantfile
-first_line_match: ^#!/.*\bruby
+first_line_match: ^#!\s*/.*\bruby\b
```

Comment:
I'll tweak this to allow `jruby` when I get home.  I don't think we should add `erb`, though - that'll most likely be some other language with a few embedded Ruby snippets and won't play nice with the Ruby syntax file.